tensorflow/compiler/tf2tensorrt/utils/funcdef_to_graphdef.cc

```diff
@@ -0,0 +1,142 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/compiler/tf2tensorrt/utils/funcdef_to_graphdef.h"
+#include "absl/strings/ascii.h"
+#include "absl/strings/str_cat.h"
+#include "tensorflow/core/common_runtime/function.h"
+#include "tensorflow/core/common_runtime/graph_optimizer.h"
+#include "tensorflow/core/framework/node_def_builder.h"
+#include "tensorflow/core/graph/algorithm.h"
+#include "tensorflow/core/graph/graph_constructor.h"
+#include "tensorflow/core/platform/logging.h"
+
+namespace tensorflow {
+namespace tensorrt {
+
+auto prefixes = IONamePrefixes();
+
+string AppendIdToNodeName(const Node* n) {
```

Comment:
The Graph-->GraphDef conversion in funcdef_to_graphdef.cc mirrors the behavior of the conversion in core/common_runtime/function.cc, I retained the uniquification of nodes (which uses the id's to rename them). I'm unsure if this is necessary or not, if we can be sure that the node names are already unique, we can remove the remapping (otherwise, moving it outside of the conversion). 
The special remapping is needed because the code identifies I/O nodes by prefix, while the method in core/common_runtime/function.cc simply renames all nodes by their id, removing prefixes.